bats/tests/helpers/defaults.bash

```diff
@@ -27,6 +27,13 @@ capturing_logs() {
     is_true "$RD_CAPTURE_LOGS"
 }
 
+########################################################################
+: "${RD_NO_MODAL_DIALOGS:=true}"
+
+suppress_modal_dialogs() {
```

Comment:
To fit in with the other predicate function (`capturing_logs`, `taking_screenshots`, etc) I think this function should be renamed:

```suggestion
suppressing_modal_dialogs() {
```

Reply:
Makes sense... if you don't want modal dialogs, you're opting for reduced functionality, and that decision should take precedence over the application deciding it has to stop for a modal dialog.